src/evaluator/functions.ts

```diff
@@ -458,6 +458,38 @@ sanity['versionOf'] = async function (args, scope, execute) {
 }
 sanity['versionOf'].arity = 1
 
+// eslint-disable-next-line require-await
+sanity['documentsOf'] = async function (args, scope, execute) {
+  if (!scope.source.isArray()) return NULL_VALUE
+
+  const value = await execute(args[0], scope)
+  if (value.type !== 'string') return NULL_VALUE
+  const baseId = value.data
+
+  // All the document belong to a bundle ID if:
```

Comment:
```suggestion
  // All the documents belong to a bundle ID if:
```

Reply:
or "A document belongs to a bundle ID if:"

Reply:
Fixed in https://github.com/sanity-io/groq-js/pull/254/commits/022c7b2f02a764aca20c74b5a8475107b2723188